webrtc.html

```diff
@@ -4587,8 +4587,10 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
             class="methods">
               <dt><dfn><code>getAlgorithm</code></dfn></dt>
               <dd>
-                <p>Returns the value of <var>keygenAlgorithm</var> passed in
-                the call to <code>generateCertificate()</code>.</p>
+                <p>Returns an AlgorithmIdentifier corresponding to the one passed in the call to
+                <code>generateCertificate()</code>, meaning an object that would produce the same
+                result by the normalization process as the original <var>keygenAlgorithm</var>
```

Comment:
Oh, nope, based on https://github.com/w3c/webrtc-pc/issues/1120#issuecomment-294765374 I removed the bit about what to do if the certificate was generated by the RTCPeerConnection.